v1/metrics.yaml

```diff
@@ -232,6 +232,59 @@ paths:
               uri: '{{options.host}}/pageviews/top/{project}/{access}/{year}/{month}/{day}'
       x-monitor: false
 
+  /unique-devices/{project}/{access-site}/{granularity}/{start}/{end}:
```

Comment:
@milimetric Ah, sorry for misunderstanding. I thought it's like "vies of the site by unique devises"..